nuxt.config.js

```diff
@@ -58,7 +58,7 @@ export default {
   /*
    ** Nuxt.js dev-modules
    */
-  buildModules: ['@nuxtjs/tailwindcss', '@nuxtjs/fontawesome', '@nuxt/image', 'nuxt-build-optimisations'],
+  buildModules: ['@nuxtjs/tailwindcss', '@nuxtjs/fontawesome', '@nuxt/image', 'nuxt-webpack-optimisations'],
```

Comment:
Nuxt renamed their build optimization library (probably bc they're gonna support Vite **and** Webpack now)